gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/plan/apikey/PlanApiKeyV4EmulationIntegrationTest.java

```diff
@@ -102,7 +102,7 @@ protected Stream<Arguments> provideWrongSecurityHeaders() {
 
     @ParameterizedTest
     @MethodSource("provideApis")
-    void should_return_200_success_with_api_key_and_subscription_on_the_api(
+    protected void should_return_200_success_with_api_key_and_subscription_on_the_api(
```

Comment:
Why do you change scope?

Reply:
as I explained in slack, I need to inherit from it to compose and override in the multiple plans scenario. you can read commit by commit to make this easier